LoRaEngine/deployment.ci.aio.template.json

```diff
@@ -112,7 +127,10 @@
               "createOptions": {
                 "HostConfig": {
                   "NetworkMode": "host",
-                  "Privileged": true
+                  "Privileged": true,
+                  "Binds":  [
+                    "$CERT_REMOTE_PATH:/var/lorastarterkit/certs/"
```

Comment:
[nit] not sure if it's worth it, but to me it was not immediately clear how the LBS picks this up. Maybe it would be worth a comment that the location needs to match what we have in `LoRaEngine\modules\LoRaBasicsStationModule\start_basestation_amd64.sh`?

Reply:
It's already documented here https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/station-credential-management/#server-authentication

I will add client authentication documentation with #877